commands/watch.go

```diff
@@ -95,3 +112,105 @@ func watch(cctx *cli.Context) error {
 	}
 	return nil
 }
+
+// LotusChainNotifier is a head event notifier that subscribes to a lens's ChainNotify method and adapts the
+// events received for use by a chain.Watcher
+// NOTE: this functionality will be probably folded into the Lotus API lens since other lenses will support more
+// direct methods of accessing new tipsets
```

Comment:
Does this mean here: https://github.com/filecoin-project/sentinel-visor/blob/master/lens/lotus/api.go#L1

or somewhere different?

Reply:
Roughly there - I am still exploring